### PR TITLE
6 consulting benefits

### PR DESCRIPTION
**Pull request by johnxussw**
#6 

- [x] create component: benefit card
    - [x] scroll animation
    - [x] style
- [x] create component: benefit list
    - [x] style
    - [x] responsive
- [x] create component: rules

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -5,8 +5,9 @@ import { Analytics } from "../components/layout/analytics";
 import { NEXT_SEO_DEFAULT } from "../next-seo.config";
 import * as gtag from "../lib/gtag";
 import "aos/dist/aos.css";
-import "../styles.css";
 import AOS from "aos";
+import "aos/dist/aos.css"
```
Comment (wicksipedia) [inline pages/_app.tsx:9]: looks like a double import
```suggestion
```

File: plugins/normalize.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const plugin = require("tailwindcss/plugin");
+
+module.exports = plugin(function ({ addBase }) {
+  addBase({
```
Comment (wicksipedia) [inline plugins/normalize.js:4]: I like this but I think it should be in the tailwind.config.js file.... similar to https://play.tailwindcss.com/uF9rN9jewc?file=config

File: tailwind.config.js [hunk 3]
```diff
@@ -144,6 +144,18 @@ module.exports = {
...
+              padding: "0.25rem",
+              borderRadius: "3px",
+              margin: "-0.25rem 1px",
+            },
+            "code::before": {
+              content: "",
+            },
+            "code::after": {
+              content: "",
+            },
```
Comment (wicksipedia) [inline tailwind.config.js:158]: I don't believe this is used anywhere. It would be good to delete it then

**Review by wicksipedia — CHANGES_REQUESTED** [review]

File: plugins/normalize.js (hunk 2 above)
Comment (johnxussw) [reply plugins/normalize.js]: Done - plugin removed

Base font has been defined in style.css

File: tailwind.config.js (hunk 3 above)
Comment (johnxussw) [reply tailwind.config.js]: Done - unused code removed

File: pages/_app.tsx (hunk 1 above)
Comment (johnxussw) [reply pages/_app.tsx:7]: Done - duplicated import removed

**Review by wicksipedia — APPROVED** [review]
lgtm

**Pull request merged**